sdk/resourcemanager/azure-resourcemanager-containerregistry/src/test/java/com/azure/resourcemanager/containerregistry/RegistryTaskTests.java

```diff
@@ -1385,6 +1387,93 @@ public void testRegistryPublicNetworkAccess() {
         Assertions.assertEquals(PublicNetworkAccess.ENABLED, registry.publicNetworkAccess());
     }
 
+    @Test
+    public void testRegistryNetworkRules() {
+        final String acrName = generateRandomResourceName("acr", 10);
+
+        // registry with default network rule settings
+        Registry registryWithDefaultNetworkRules = registryManager.containerRegistries()
+            .define(acrName)
+            .withRegion(Region.US_EAST)
+            .withNewResourceGroup(rgName)
+            .withPremiumSku()
+            .create();
+
+        Assertions.assertEquals(SkuTier.PREMIUM, registryWithDefaultNetworkRules.sku().tier());
+        Assertions.assertEquals(PublicNetworkAccess.ENABLED, registryWithDefaultNetworkRules.publicNetworkAccess());
+        Assertions.assertTrue(registryWithDefaultNetworkRules.canAccessFromTrustedServices());
```

Comment:
So canAccessFromTrustedServices is default true if public enabled.

What's the default when public disabled? Is it false?

Reply:
I believe it's still true. I'll add a test case.

Reply:
Verified. Public network access doesn't affect this setting.